src/test/ui/associated-type/associated-type-projection-from-multiple-supertraits.stderr

```diff
@@ -20,19 +35,28 @@ LL |     type Color;
    |     ----------- ambiguous `Color` from `Box`
 ...
 LL | fn dent_object<COLOR>(c: dyn BoxCar<Color=COLOR>) {
-   |                                     ^^^^^^^^^^^ ambiguous associated type `Color`
+   |                              ^^^^^^^^^^^^^^^^^^^ ambiguous associated type `Color`
+   |
+help: use fully qualified syntax to disambiguate
+   |
+LL | fn dent_object<COLOR>(c: dyn <BoxCar as Box>::Color) {
+   |                              ^^^^^^^^^^^^^^^^^^^^^^
+help: use fully qualified syntax to disambiguate
+   |
+LL | fn dent_object<COLOR>(c: dyn <BoxCar as Vehicle>::Color) {
+   |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This suggestion is not exactly what we would like to provide for this case, but this should have its own error code, as the solution is a bigger change than what we can demonstrate using structured suggestions.